public/index.html

```diff
@@ -24,10 +20,8 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
   </head>
   <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
```

Comment:
This boilerplate can be useful to those who don't have JS enabled in their browsers (although that is very rare)